aplib.net-demo/Assets/Scripts/GameManager.cs

```diff
@@ -0,0 +1,28 @@
+using UnityEngine;
+
+public class GameManager : MonoBehaviour
+{
+    private static GameManager _instance;
+
+    public static GameManager Instance
+    {
+        get
+        {
+            if (_instance == null) Debug.LogError("Game Manager is null, first awake it!");
+            
+            return _instance;
+        }
+    }
+
+    private void Awake()
+    {
+        // Setup singleton
+        if (_instance != null && _instance != this) {
+            Destroy(gameObject);
+        } else {
+            _instance = this;
+        }
```

Comment:
```suggestion
        if (_instance != null && _instance != this)
            Destroy(gameObject);
        else
            _instance = this;
```
Can probably remove this? Not sure what the rules for this are but saves some screen space in the long run I suppose

Reply:
I don't know either but I will remove them.